OpenDreamClient/Interface/DMF/DMFParser.cs

```diff
@@ -221,14 +222,27 @@ public sealed class DMFParser(DMFLexer lexer, ISerializationManager serializatio
                 if (!Check(TokenType.Value) && !Check(TokenType.Attribute))
                     Error($"Invalid attribute value ({valueText})");
             } else if (!Check(TokenType.Value))
-                if(Check(TokenType.Semicolon)) //thing.attribute=; means thing.attribute=empty string
+                if(Check(TokenType.Semicolon) || Check(TokenType.EndOfFile)) //thing.attribute=; means thing.attribute=empty string
                     valueText = "";
                 else
                     Error($"Invalid attribute value ({valueText})");
+            else if (Check(TokenType.Ternary)) {
+                List<DMFWinSet> ifValues = new();
+                List<DMFWinSet> elseValues = new();
```

Comment:
In BYOND are there allowed to be multiple ifValues or more than 2 elseValues?

Reply:
Yup, you can do `condition=value?thing1=1;thing2=2:thing1=0;thing2=0`

Reply:
That's still only 1 condition and 2 branches

Reply:
Oh I see what you mean, sorry this var is poorly named. The ifValues are the statements evaluated when the condition is true.

Reply:
renamed to TrueStatements and FalseStatements, plus code doc for extra clarity